core/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.9.10"
+__version__ = "0.9.11"
```

Comment:
```suggestion
__version__ = "0.10.0"
```

Reply:
Thanks - added in https://github.com/nwsde/nwsde-azuretre/commit/ad5ac13907869df2efb262dd0a6b993b7df2b504